aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/plugin/target/TargetImpl.java

```diff
@@ -169,8 +177,11 @@ private Target getTarget(String path, boolean createIfMissing) {
         if (StringUtils.isBlank(path)) {
             return null;
         }
-        if (path.contains(DialogConstants.PATH_SEPARATOR)) {
-            Queue<String> pathChunks = Pattern.compile("/").splitAsStream(path).collect(Collectors.toCollection(LinkedList::new));
+        String effectivePath = StringUtils.strip(path, DialogConstants.DOUBLE_QUOTE);
+        boolean isEscaped = !StringUtils.equals(path, effectivePath);
+
+        if (!isEscaped && effectivePath.contains(DialogConstants.PATH_SEPARATOR)) {
+            Queue<String> pathChunks = Pattern.compile("/").splitAsStream(effectivePath).collect(Collectors.toCollection(LinkedList::new));
```

Comment:
(cosmetics) Pattern.compile(PATH_SEPARATOR)?

Reply:
Exactly, thank you